packages/react-liveness/src/components/FaceLivenessDetector/shared/LivenessIconWithPopover.tsx

```diff
@@ -5,9 +5,10 @@
 
 import * as React from 'react';
 
-import { Flex } from '@aws-amplify/ui-react';
+import { Button, Flex } from '@aws-amplify/ui-react';
 import { AlertIcon, useThemeBreakpoint } from '@aws-amplify/ui-react/internal';
 import { LivenessClassNames } from '../types/classNames';
+import { defaultLivenessDisplayText } from '../displayText';
```

Comment:
Sorry I didn't really explain this part; we won't be able to import directly from this because these are just our _default_ displayText; our customers can optionally provide their own object of displayText matching our dictionary then and we merge them together at the [root of the component](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/FaceLivenessDetectorCore.tsx#L44).

I think we'll need to add new strings for the aria-labels (for Button at least, and maybe for the aria-label on the dialog element), so you'll need to update displayText.ts with the strings/type updates, and then update what strings as props we pass down to the [DefaultPhotosensitiveWarning](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react-liveness/src/components/FaceLivenessDetector/StartLiveness/StartLiveness.tsx#L42) component in order to use them in your changes below.